src/ServiceControl.Plugin.Nsb6.SagaAudit.AcceptanceTests/When_saga_changes_state.cs

```diff
@@ -0,0 +1,118 @@
+﻿namespace ServiceControl.Plugin.Nsb6.SagaAudit.AcceptanceTests
+{
+    using System;
+    using System.Configuration;
+    using System.Threading.Tasks;
+    using EndpointPlugin.Messages.SagaState;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_saga_changes_state : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_send_result_to_service_control()
+        {
+            var context = await Scenario.Define<Context>(c => { c.Id = Guid.NewGuid(); })
+                .WithEndpoint<Sender>(b => b.When(session => session.SendLocal(new StartSaga
+                {
+                    DataId = Guid.NewGuid()
+                })))
+                .WithEndpoint<FakeServiceControl>()
+                .Done(c => c.WasCalled && c.TimeoutReceived)
```

Comment:
As of right now it just times out, when I disable the startup task checking for the SC queue.